addons/mail/static/src/components/thread_view/thread_view.xml

```diff
@@ -8,57 +8,65 @@
+                                    Click here to retry
+                                </button>
+                            </div>
+                        </t>
+                        <t t-elif="props.hasComposer">
+                            <div class="o-autogrow"/>
+                        </t>
+                        <t t-if="props.hasComposer">
+                            <Composer
+                                class="o_ThreadView_composer"
+                                attachmentsDetailsMode="props.composerAttachmentsDetailsMode"
+                                composerLocalId="threadView.thread.composer.localId"
+                                hasCurrentPartnerAvatar="props.hasComposerCurrentPartnerAvatar"
+                                hasSendButton="props.hasComposerSendButton"
+                                hasThreadTyping="props.hasComposerThreadTyping"
+                                isCompact="(threadView.thread.model === 'mail.channel' and threadView.thread.mass_mailing) ? false : undefined"
+                                isDoFocus="props.isDoFocus"
+                                showAttachmentsExtensions="props.showComposerAttachmentsExtensions"
+                                showAttachmentsFilenames="props.showComposerAttachmentsFilenames"
+                                textInputSendShortcuts="threadView.textInputSendShortcuts"
+                                t-ref="composer"
+                            />
+                        </t>
                     </div>
-                </t>
-                <t t-elif="props.hasComposer">
-                    <div class="o-autogrow"/>
-                </t>
-                <t t-if="props.hasComposer">
-                    <Composer
-                        class="o_ThreadView_composer"
-                        attachmentsDetailsMode="props.composerAttachmentsDetailsMode"
-                        composerLocalId="threadView.thread.composer.localId"
-                        hasCurrentPartnerAvatar="props.hasComposerCurrentPartnerAvatar"
-                        hasSendButton="props.hasComposerSendButton"
-                        hasThreadTyping="props.hasComposerThreadTyping"
-                        isCompact="(threadView.thread.model === 'mail.channel' and threadView.thread.mass_mailing) ? false : undefined"
-                        isDoFocus="props.isDoFocus"
-                        showAttachmentsExtensions="props.showComposerAttachmentsExtensions"
-                        showAttachmentsFilenames="props.showComposerAttachmentsFilenames"
-                        textInputSendShortcuts="threadView.textInputSendShortcuts"
-                        t-ref="composer"
-                    />
-                </t>
+                    <t t-if="threadView.thread and threadView.thread.model === 'mail.channel' and threadView.hasMemberList">
+                        <div class="o_ThreadView_coreChannelMemberListSeparator"/>
```

Comment:
Would remove the prefix "core" in the part name, it's doesn't look important concern and resulting name would be easier to catch (feels less technical)

Reply:
This is the separator between "core" and "member list" so I prefer to keep both names in there (but I agree it's a bit verbose)